apps/reference/docs/guides/auth/auth-mfa.mdx

```diff
@@ -0,0 +1,570 @@
+---
+id: auth-mfa
+title: 'Multi-Factor Authentication'
+description: Add an additional layer of security to your apps with Supabase Auth multi-factor authentication.
+---
+
+import { Mermaid } from 'mdx-mermaid/Mermaid';
+
+:::note
+Multi-Factor Authentication is in early access preview only.
+:::
+
+Multi-factor authentication (MFA) or sometimes also called two-factor
+authentication (2FA) is a way to add an additional layer of security to your
+application by requiring users to verify their identity through one or more
+additional verification steps.
+
+It is considered a best practice for all but the simplest of applications.
+
+Users who use weak passwords or have compromised social login accounts are
+prone to malicious account takeovers. These can be prevented by using
+multi-factor authentication because they require the user to provide proof of
+both of these:
```

Comment:
```suggestion
Users with weak passwords or compromised social login accounts are
prone to malicious account takeovers. These can be prevented with
MFA because they require the user to provide proof of
both of these:
```